governator-core/src/main/java/com/netflix/governator/conditional/AllOfConditional.java

```diff
@@ -0,0 +1,27 @@
+package com.netflix.governator.conditional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.inject.Injector;
+
+/**
+ * Conditional that is true if and only if all child conditionals are true
+ */
+public class AllOfConditional implements Conditional {
+    private final List<Conditional> children;
+    
+    public AllOfConditional(List<Conditional> children) {
```

Comment:
should be able to take any Iterable or Collection as input, and add a varargs constructor too?